chezbetty/locale/fa/LC_MESSAGES/betty.po

```diff
@@ -0,0 +1,736 @@
+#
+# Persian translations for PACKAGE package
+# This file is distributed under the same license as the PACKAGE package.
+# Amir Rahmati <amir@rahmati.com>, 2015.
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE 1.0\n"
+"POT-Creation-Date: 2015-04-07 18:36-0400\n"
+"PO-Revision-Date: 2015-04-07 22:17-0400\n"
+"Last-Translator: Amir Rahmati <amir@rahmati.com>\n"
+"Language-Team: Persian\n"
+"Language: fa\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Lingua 3.9\n"
+
+#. Default: User is not enabled. Please contact ${email}.
+#: chezbetty/views.py:136
+msgid "user-not-enabled"
+msgstr "کاربر‫-‬فعال‫-‬نشده"
+
+#. Default: Failed to read M-Card. Please try swiping again.
+#: chezbetty/views.py:173
+msgid "mcard-error"
+msgstr "ایراد‫-‬ام‌کارد"
+
+#: chezbetty/views_user.py:165
+msgid "Deposit amount must be greater than $0.00. Card NOT charged."
+msgstr "مبلغ واریز شده باید بیشتر از صفر باشد‫.‬ کارت شارژ نشد‫.‬"
+
+#: chezbetty/templates/deposit_edit.jinja2:3
+#: chezbetty/templates/deposit_edit.jinja2:4
+msgid "Edit Deposit"
+msgstr "ویرایش واریز"
+
+#: chezbetty/templates/deposit_edit.jinja2:16
+msgid "New Deposit Amount"
+msgstr "مقدار جدید واریز"
+
+#: chezbetty/templates/deposit_edit.jinja2:28
+#: chezbetty/templates/deposit.jinja2:28
+msgid "Deposit To Your Account"
+msgstr "واریز پول به حساب"
+
+#: chezbetty/templates/deposit_edit.jinja2:30
+#: chezbetty/templates/deposit.jinja2:30
+msgid "Deposit To %(pool_name)s Pool"
+msgstr "واریز به حساب مشترک %(pool_name)"
```

Comment:
This needs to be `%(pool_name)s`, not `%(pool_name)` [sorry, the translation syntax kind of sucks]